CONTRIBUTING.md

```diff
@@ -29,6 +30,37 @@ First of all, thanks for taking the time to contribute! This project can only gr
   <li>"icon" (lowercase) refers specficially to the font icon versions of the Icons.</li>
 </ol>
 
+<hr>
+<h2 id='iconRequirements'>What Icons Do We Accept?</h2>
+<p>Devicon only accepts Icons of development languages and tools. </p>
+<p><b>Development</b> refers to programming or programming related jobs.</p>
+<p><b>Tools</b> can be software, OS, services, etc... that helps with development. It must be specifically related to development (so softwares like Microsoft Word or Google Calendar won't be accepted since it's too general).</p>
```

Comment:
```suggestion
<p><b>Tools</b> can be software, OS, services, etc. that helps with development. It must be specifically related to development (so software like Microsoft Word or Google Calendar won't be accepted since it's too general).</p>
```